core-bundle/src/Resources/contao/library/Contao/Email.php

```diff
@@ -438,32 +434,13 @@ public function sendTo()
 		}
 
 		// Sender
-		if ($this->strSenderName != '')
-		{
-			$this->objMessage->setFrom(array($this->strSender=>$this->strSenderName));
-		}
-		else
-		{
-			$this->objMessage->setFrom($this->strSender);
-		}
+		$this->objMessage->from(new Address($this->strSender, $this->strSenderName ?? ''));
 
 		// Set the return path (see #5004)
-		$this->objMessage->setReturnPath($this->strSender);
+		$this->objMessage->returnPath($this->strSender);
 
 		// Send the e-mail
-		$intSent = $this->objMailer->send($this->objMessage, $this->arrFailures);
-
-		// Log failures
-		if (!empty($this->arrFailures))
-		{
-			System::log('E-mail address rejected: ' . implode(', ', $this->arrFailures), __METHOD__, $this->strLogFile);
-		}
-
-		// Return if no e-mails have been sent
-		if ($intSent < 1)
-		{
-			return false;
-		}
+		$this->objMailer->send($this->objMessage);
```

Comment:
#### Note

I am not actually sure which kind of "failures" are supposed to be catched here. From what I can tell `Swift_Mailer::send` only catches `Swift_RfcComplianceException`, e.g. for a malformed email address. _But:_ that exception is already thrown way earlier when executing `$this->objMessage->setTo($arrRecipients);` (in the original version). This exception is actually being catched in the `contao/newsletter-bundle` for example.

Since it seems to be superfluous and since the Symfony Mailer component does not provide a similar functionality, I did not implement any special handling here.

Reply:
So the Symfony Mailer allows us to send a mail to e.g. `invalid;;;`?

Reply:
No, this will throw an `RfcComplianceException` here:

```php
$this->objMessage->to('invalid;;;');
```

And the same happens with Swiftmailer. The `contao/newsletter-bundle` catches that exception here: 

https://github.com/contao/contao/blob/ebe47531ee32846d8dadcb428b3621530baa9090/newsletter-bundle/src/Resources/contao/classes/Newsletter.php#L386-L394

I changed the exception handling in this PR accordingly.

Reply:
Then I think it is fine to remove the failure handling here. 👍